Kernel/Interrupts/InterruptManagement.cpp

```diff
@@ -117,7 +117,7 @@ RefPtr<IRQController> InterruptManagement::get_responsible_irq_controller(u8 int
     if (m_interrupt_controllers.size() == 1 && m_interrupt_controllers[0]->type() == IRQControllerType::i8259) {
         return m_interrupt_controllers[0];
     }
-    for (auto irq_controller : m_interrupt_controllers) {
+    for (const auto& irq_controller : m_interrupt_controllers) {
```

Comment:
I *think* it's idiomatic Serenity style to leave out the `const` in `const auto`, because the compiler can infer it.

Reply:
Is it? 

> $ rg "const auto&" | wc -l
> 173

Reply:
Another cultured `rg` user, I see.
If you compare the counts instead...
```
$ rg --pcre2 '(?!=const )auto&' | wc -l
4390

# vs

rg --pcre2 'const auto&' | wc -l
174
```

Reply:
🤷 sure, "fixed".

Reply:
I don't mind either way, just thought I'd add that bit lol